src/bdc/steps/google_places.py

```diff
@@ -72,14 +77,31 @@ def run(self) -> pd.DataFrame:
         return self.df
 
     def finish(self) -> None:
-        pass
+        p_matches = (
+            self._df["google_places_place_id_matches_phone_search"].sum()
```

Comment:
I get an error here and the upcoming lines 86 and 87, I believe you meant self._df["place_id_matches_phone_search"], right ?

Reply:
Could you share the error message? As the df is accessed I need to prepend the google_places_ prefix to any column from that step.

Reply:
Ah i see.. You'd have to run the Phonenumber Validation first. The Google Step is now depending on the `number_formatted` field. In your current config, the google step is not executed, as this column is missing. Still the `finish()` method of the step is executed because of the way the pipeline framework is built. Ofc if `run()` is not executed because `verify()` failed, the column won't exist in the dataframe.

Reply:
I'll have to introduce a check to `finish()` or change the sequence in `pipeline.py`..

Reply:
I see, otherwise great work!